dev/devicelab/bin/tasks/plugin_test_ios.dart

```diff
@@ -8,14 +8,6 @@ import 'package:flutter_devicelab/tasks/plugin_tests.dart';
 Future<void> main() async {
   await task(
     combine(<TaskFunction>[
-      PluginTest('ios', <String>['-i', 'objc', '--platforms=ios']).call,
-      PluginTest('ios', <String>['-i', 'swift', '--platforms=ios']).call,
-      // Test that app builds with Flutter as a transitive dependency.
-      PluginTest('ios', <String>[
-        '-i',
-        'objc',
-        '--platforms=ios',
-      ], cocoapodsTransitiveFlutterDependency: true).call,
```

Comment:
I think we still want to test `cocoapodsTransitiveFlutterDependency`.
Also this deletes testing plugins without `dartOnlyPlugin`.
```suggestion
      PluginTest('ios', <String>['--platforms=ios']).call,
      PluginTest('ios', <String>['--platforms=ios'], cocoapodsTransitiveFlutterDependency: true).call,
```